crates/stackable-versioned-macros/src/codegen/container/struct.rs

```diff
@@ -283,18 +345,30 @@ impl Struct {
                     .singular
                     .as_ref()
                     .map(|s| quote! { , singular = #s });
+
                 let plural = kubernetes_options
                     .plural
                     .as_ref()
                     .map(|p| quote! { , plural = #p });
+
                 let namespaced = kubernetes_options
                     .namespaced
                     .then_some(quote! { , namespaced });
                 let crates = kubernetes_options.crates.to_token_stream();
-                let status = kubernetes_options
-                    .status
-                    .as_ref()
-                    .map(|s| quote! { , status = #s });
+
+                let status = if multiple_versions {
+                    let status_ident = format_ident!(
+                        "{struct_ident}Status",
+                        struct_ident = self.common.idents.kubernetes.as_ident()
+                    );
+                    Some(quote! { , status = #status_ident })
+                } else {
```

Comment:
I would prefer to either wait with the merge until the Status structure is proven to be working or comment this code out or put it behind an opt-in

Reply:
Commented out in c448ed2.